openapi.yaml

```diff
@@ -1,190 +1,480 @@
+        - parks
+      description: Delete park by unique ID
+      operationId: deleteParkById
+      parameters:
+        - $ref: '#/parameters/parkIdPath'
+      responses:
+        '204':
+          description: Park successfully deleted
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
+  /owners:
     get:
-      summary: Info for a specific pet
+      summary: Lists all park owners
+      tags:
+        - owners
+      description: Get all park owners
+      operationId: getOwners
+      responses:
+        '200':
+          description: Successful Response
+          schema:
+            $ref: '#/definitions/OwnersResult'
+        '500':
+          $ref: '#/responses/500'
+    post:
+      summary: Creates a new park owner account
       tags:
-        - pets
-      description: Get pet by unique ID
-      operationId: getPetById
+        - owners
+      description: Post a new park owner
+      operationId: postOwner
       parameters:
-        - $ref: '#/parameters/authorization'
-        - name: id
-          in: path
-          description: Unique Pet ID
-          required: true
-          type: string
+      - $ref: '#/parameters/ownerPostBody'
+      responses:
+        '201':
+          description: Successfully created new owner
+          headers:
+            Location:
+              type: string
+              description: Location of newly created owner
+          schema:
+            $ref: '#/definitions/OwnerResult'
+        '400':
+          $ref: '#/responses/400'
+        '409':
+          description: invalid type, or ownerName already in use
+          schema:
+            $ref: '#/definitions/ErrorResult'
+        '500':
+          $ref: '#/responses/500'
+  /owners/{ownerId}:
+    get:
+      summary: gets the account of a park owner by ID
+      tags:
+        - owners
+      description: gets park owner by ID
+      operationId: getOwnerById
+      parameters:
+      - $ref: '#/parameters/ownerIdPath'
+      responses:
+        '200':
+          description: Successful Response
+          schema:
+            $ref: '#/definitions/OwnerResult'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
+    patch:
+      summary: Updates the account of a park owner by ID
+      tags:
+        - owners
+      description: patch park owner by ID
+      operationId: updateOwnerById
+      parameters:
+      - $ref: '#/parameters/ownerIdPath'
+      - $ref: '#/parameters/ownerPatchBody'
       responses:
         '200':
-          description: Successful response
+          description: Successful Response
           schema:
-            $ref: '#/definitions/PetResult'
+            $ref: '#/definitions/OwnerResult'
+        '400':
+          $ref: '#/responses/400'
         '404':
-          description: PetResource not found
+          $ref: '#/responses/404'
+        '409':
+          description: id or type can't be changed
```

Comment:
Couldn't this also occur if trying to patch an owner with a new name that's already in use?